hazelcast-spring/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -175,11 +175,6 @@
                 </hz:ssl>
                 <hz:socket-interceptor enabled="false" class-name="com.hazelcast.spring.DummySocketInterceptor"
                                        implementation="dummySocketInterceptor"/>
-                <hz:symmetric-encryption enabled="true"
```

Comment:
Can you set to `enabled=false` and still assert it was properly parsed in `testNetworkConfig()`?

Reply:
Yes. It's also a possibility. I'll update it.